bin/delete_workers.sh

```diff
@@ -13,9 +16,4 @@ if kubectl get sts redis-master &> /dev/null; then
   done
 fi
 
-# Workers now handle deleting themselves from redis on shutdown. This will happen faster than when
-# the redis keys are deleted.
-echo "Deleting worker jobs..."
-kubectl delete job -l run.beam.cloud/role=worker
-
 wait
```

Comment:
This was a changed before because of the worker not found error. Its being reverted now.